### PR TITLE
[CHORE] Initialise QuickReplies App 

### PR DESCRIPTION
**Pull request by VipinDevelops**
### Issues 
#1 

###  Acceptance Criteria

- [x] Initialise RC App
- [x]  Add Icon
- [x]  Add bug form
- [x]  Add feat form
- [x]  Add PR template
- [x]  Setup lint & prettier

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"EditorConfig.editorconfig",
+		"eamodio.gitlens",
+		"eg2.vscode-npm-script",
+		"wayou.vscode-todo-highlight",
+		"minhthai.vscode-todo-parser",
+		"ms-vscode.vscode-typescript-tslint-plugin",
+		"rbbit.typescript-hero"
```
Comment (hugocostadev) [inline .vscode/extensions.json:9]: let's keep only te essentials, IMO would be editorconfig, eslint and prettier, right?

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		"quotemark": [true, "single"],
+		"max-line-length": [
+			true,
+			{
+				"limit": 160,
+				"ignore-pattern": "^import | *export .*? {"
+			}
+		]
+	}
+}
```
Comment (hugocostadev) [inline tslint.json:18]: is this necessary?  Why don't just use eslint and tsconfig combination?

**Review by hugocostadev — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 1 above)
Comment (VipinDevelops) [reply .vscode/extensions.json:9]: Sure, this was created by default when we made a new RC app, so I just let it stay.

File: tslint.json (hunk 2 above)
Comment (VipinDevelops) [reply tslint.json:18]: Yeah We can just use them, but I am trying to follow the same Standard as other popular RC apps

**Pull request merged**